### DISCUSSION TITLE
在再很重要嗎

### DISCUSSION THREAD

**Discussion by zhilin93517** [1101-GS4719-C/Python-Community-GS4719-C#2161] [General]
「你再做什麼?」、「可以在說一次嗎」、「你不因該這麼做」
這些都是在網路上常常可以看到的錯別字，若有人在這種留言下方糾正，反而可能收到回覆「看得懂就好」這種答案。
你們認為這種錯字是可以被允許的嗎，你們喜歡在網路上看到這種有錯別字但不會影響語意的文句嗎?

**Comment by Jinyuu17** [comment]
「在」有點像是英文的現在進行式：
我正在吃飯 I am eating.

「再」表示重複或將來的動作，比較像是未來式
我會再做一次 I will do it again.

這樣應該可以比較知道為什麼很重要了嗎，時態都不太一樣了。它們不僅影響句子的準確性，還會影響理解者對內容的認識。

**Comment by 112121py** [comment]
我覺得看場合跟人欸，有些人就真的沒辦法忍受錯字，然後還不改，或是標點符號全形半形等等，個人是不太在乎，但看到的時候還是會在心裡幫他糾正一下xd
然後日常如果習慣正確使用字的話，我覺得在正式的場合上也比較不容易出錯，至少不會有太大的壞處啦

**Comment by gigitilto** [comment]
我不太接受這種很常見的錯字，但我覺得這是自動選字的問題，如果自動選字可以再聰明一點，就不會有這種讓人看得不舒服的錯字了。

**Comment by Mminniehuang** [comment]
我覺得如果像是跟朋友傳訊息，你一言我一語即時聊天，一點點錯誤對我個人來說無傷大雅，因為確實打字很快或是手滑按錯是可以理解的，我自己也有時候會打錯（不過我通常還是會立刻改正罰寫幾次傳給對方，順便看可不可以試圖改善我的自動選字😂）
但我覺得其他時候還是嚴謹一點比較好，畢竟工作、研究等等，有錯字或訊息影響可能會比想像中影響更嚴重，有時候不是「看得懂就好」可以解決的。

**Comment by kzx-creat** [comment]
在網路上看到別人在再不分都會心癢癢......... 但果是私底下聊天錯字，我就不太在意。畢竟有時候聊得起勁，瘋狂打字也沒時間管那不影響理解的小錯字。

**Comment by wcyeee** [comment]
我覺得除了某些人真的看了就會渾身不舒服，不然這個其實好像不是一個很嚴重影響理解的問題XD 就像之前一些圖片是寫上一句話，我們可以一看就知道唸出來這句話是什麼，但是認真看之後才發現文字都被打亂了，所以我覺得這種小小的錯別字好像不會很影響溝通。（但某些字真的不能錯 意思會整個不一樣）（又或是工作等等這種正式場合 就算是小小的錯別字也不要錯比較好）

**Comment by christaaah** [comment]
雖然大部分的都是沒注意有錯誤就送出，但確實有少部分人是真的在再分不清，前一種的話我覺得偶爾錯一兩次不會怎麼樣，但是如果變成習慣那就比較嚴重了，如果是平常和朋友家人聊天就算了，但是正式文件就不容許有這種錯誤發生，所以還是從平常就習慣比較好。

**Comment by qiu0109** [comment]
跟朋友用訊息聊天的話，通常打字會很快，同時手機又有自動選字，因此打錯「在」跟「再」我認為在所難免，相信應該很多人都有打錯過。
反倒是「應該」打錯成「因該」會比較讓人感到煩躁，如果是在正式文件中，會顯得很不專業。

**Comment by lai1008** [comment]
跟朋友用訊息聊天通常打字會很快，因此很常打錯「在」跟「再」我認為無法難免，我相信大部分人都有打錯過。
但只要對方的懂就好。

**Comment by Tchang323** [comment]
我覺得 對於正在學習中文的人很重要XD 但對我們一般母語人士，我認為在生活中的溝通，只要看得懂就好(考試例外

**Comment by ed112601513** [comment]
偶爾出現一些錯字我覺得在所難免，如果是在和朋友或家人聊天，我不會太在意這些錯別字的問題，畢竟語言本身就只是為了溝通，但如果是在嚴謹一點的場合，比如:報告、專案、論文等，這些給別人看到且閱讀的場合時就應該要重視錯別字，以免他人誤解或翻譯有誤等情況發生。

**Comment by 113607514** [comment]
我覺得很重要因為文字創造本來就有該字的用法，如果亂用字的話我覺得會天下大亂，的和地也亂用，到最後 後人分不出來，那文字就失去意義了

**Comment by poyen0806** [comment]
我覺得在再很重要，畢竟兩個字的意義還是不同，在某些情境會讓人產生誤解，就像從口中說出兩個讀音相近的字時，若講錯發音，則可能使人誤解，所以我覺得在再很重要

**Comment by forestdew507** [comment]
雖然這樣說很模糊，但我看關於在再的誤用真的是看體質，有或者有些文字工作者應該會比較敏感。我個人的話如果是看到留言是不會去糾正啦，自己打字的時候我會特別留意，但有時候送出後懶得改也常發生哈

**Comment by yuan3x2** [comment]
我認為要看場合，重要場合如果出現錯字就容易給人一種隨便、不重視的感覺，但日常生活中如果不小心打錯我是認為沒關係，可以提醒但沒必要像一些糾錯字警察一樣一直謾罵。

**Comment by Boying12** [comment]
隨著網路交流的普及，年輕人和學生們在網上進行學習和互動，而正確的書寫習慣有助於培養良好的語言能力。當我們習慣於錯別字的存在，可能會影響到我們在正式場合的表達能力，甚至在未來的工作中造成不必要的誤解。

**Comment by yue083139** [comment]
我認為錯別字的糾正其實還挺重要的，因為有時一個錯別字真的會很大程度地影響我們對於文章語意的理解，雖然有時因為打字的緣故而不小心疏忽是可以理解，但若是沒有重視這個問題而養成習慣的話，可能會導致在真正正式的場合也會出現錯誤。

**Comment by Taoyi-Chiang** [comment]
我自己覺得還好耶，可能要從基本能力討論。
比方說如果是基本能力不好，那可能是比較嚴重的，需要分辨清楚。
但如果是選字的問題，那應該相對就比較沒關係吧？
個人淺見wwwwww
